exporter/lokiexporter/README.md

```diff
@@ -0,0 +1,62 @@
+# Loki Exporter
+
+Exports data via HTTP to [Loki](https://grafana.com/docs/loki/latest/). By default, this exporter requires TLS and
+offers queued retry capabilities.
+
+Supported pipeline types: logs
+
+## Getting Started
+
+The following settings are required:
+
+- `endpoint` (no default): The target URL to send Loki log streams to (
+  e.g.: https://loki.example.com:3100/loki/api/v1/push).
+
+
+- `allowed_attributes_for_stream_labels` (no default): List of allowed attributes to be added as labels to Loki log 
+  streams. This is a safety net to help prevent accidentally adding dynamic labels that may significantly increase 
+  cardinality, thus having a performance impact on your Loki instance. See the 
+  [Loki label best practices](https://grafana.com/docs/loki/latest/best-practices/current-best-practices/) page for 
+  additional details on the types of labels you may want to associate with log streams.
+
+The following settings can be optionally configured:
+
+- `insecure` (default = false): When set to true disables verifying the server's certificate chain and host name. The
+  connection is still encrypted but server identity is not verified.
+- `ca_file` (no default) Path to the CA cert. For a client this verifies the server certificate. Should only be used if
+  `insecure` is set to false.
+- `cert_file` (no default) Path to the TLS cert to use for TLS required connections. Should only be used if `insecure`
+  is set to false.
+- `key_file` (no default) Path to the TLS key to use for TLS required connections. Should only be used if `insecure` is
+  set to false.
+
+
+- `timeout` (default = 30s): HTTP request time limit. For details see https://golang.org/pkg/net/http/#Client
+- `read_buffer_size` (default = 0): ReadBufferSize for HTTP client.
+- `write_buffer_size` (default = 512 * 1024): WriteBufferSize for HTTP client.
+
+
+- `headers` (no default): Name/value pairs added to the HTTP request headers. Loki by default uses the "X-Scope-OrgID"
+  header to identify the tenant the log is associated to.
+
+Example:
+
+```yaml
+loki:
+  endpoint: https://loki.example.com:3100/loki/api/v1/push
+  allowed_labels:
+    - app
```

Comment:
These are supposed to be attribute names I think so using realizstic semantic convention names would be clearer.

Reply:
Addressed